### DISCUSSION TITLE
Update.fields() not working?

### DISCUSSION THREAD

**Discussion by guidorietbroek** [O365/python-o365#985] [General]
First of all great library! Tx!

I'm trying to set some metadata fields and using this code, where `updates` is a dictionary.

`
account.sharepoint().get_site('DOMAIN.sharepoint.com:/sites/SITE').get_list_by_name('Documents').get_item_by_id(1).update_fields(updates)
`

I get no errors, but when looking at the fields, nothing changed. I also see that it's possible to use save_updates() on an item. I can't use this on the update_fields() method.

Do I miss something?

**Comment by guidorietbroek** [comment]
So, I checked the code and it looks like `update_fields()` isn't returning anything but is creating some kind of track changes. The `save_updates()` method is actually using this track changes. But on what do you call this method because `update_fields()` returns a NoneType.

**Reply by jtweeder** [reply]
Hey @guidorietbroek , the `save_updates()` method is called on the sharepoint list item you are wanting to update.

Hopefully the example in these [docs](https://o365.github.io/python-o365/latest/usage/sharepoint.html) at the bottom will help.

**Comment by guidorietbroek** [comment]
This is great! Thanks. I wasn’t aware of these docs. Otherwise I would definitely not ask this question 😃
